app/assets/stylesheets/article-show.scss

```
@@ -1406,20 +898,18 @@ article {
 }
 
 .show-comments-header {
-  width: 800px;
-  max-width: 90%;
+  // width: 800px;
+  // max-width: 90%;
   margin: auto;
   margin-bottom: -50px;
   text-align: center;
   font-family: var(--ff-default);
 }
 
```

```suggestion
.primary-sticky-nav {
  display: none;
}
.primary-sticky-nav {
  display: none;
}
.article-actions {
  display: none;
}
```

Basically some of the old pages will look weird with the old style removed but old HTML still present. This will hide those elements for the edge cases. There will be some light confusion, but better than styles being off.